docs/apache-airflow/administration-and-deployment/logging-monitoring/logging-tasks.rst

```diff
@@ -57,17 +57,21 @@ In addition, you can supply a remote location to store current logs and backups.
 Writing to task logs from your code
 -----------------------------------
 
+Airflow uses standard Python `logging <https://docs.python.org/3/library/logging.html>`_ framework to
+write logs, but for duration of task, rotl logger is configured to write to task log.
```

Comment:
```suggestion
write logs, and for duration of task, root logger is configured to write to task log.
```

Reply:
it's a very subtle difference but i just suggest `and` because ... the "root logger configured" point isn't in conflict with "airflow uses standard logging" -- the two points are just true together